changelogs/fragments/51134-docker_swarm_service-change-on-updated-image.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "docker_swarm_service - Trigger a change when an image changes in a service."
```

Comment:
You should add a comment (though not here, probably somewhere in the module's documentation, maybe in the `notes:` section?) that this only works with a new enough docker-py and docker API version.

Reply:
Added a `notes:` list to the module description.